src/api/Feed.js

```diff
@@ -184,14 +185,73 @@ export const getParsedFileActivitiesResponse = (response?: { entries: FileActivi
                     return { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
                 }
 
+                case FILE_ACTIVITY_TYPE_VERSION: {
+                    const versionsItem = { ...source[FILE_ACTIVITY_TYPE_VERSION] };
+
+                    versionsItem.type = 'file_version';
+                    if (versionsItem.action_by) {
+                        const collaborators = {};
+
+                        if (versionsItem.action_by.length === 1) {
+                            versionsItem.uploader_display_name = versionsItem.action_by[0].name;
+                        }
+
+                        versionsItem.action_by.map(collaborator => {
```

Comment:
it's a little odd to use `map` without intending to use the returned array. I think this might be a good use case for `reduce`:

```
versionsItem.collaborators = versionsItem.action_by.reduce( ... , {});
```

Reply:
Leaning for keeping it for now since we've used `map` in a few other place. Going through another round of manual testing for the logic change also is a big time sink.

Reply:
semantically it feels a little wrong to use a `map` in this case over something like a `forEach` since the goal is to populate a separate object instead of using the returned array

with the reduce it could be:
```
versionsItem.collaborators = versionsItem.action_by.reduce((collaborators, collaborator) =>
    ({ ...collaborators, [collaborator.id]: { ...collaborator } }), {});
```

although you could leave it as-is if the current implementation is easier to read